app/src/main/java/com/jerboa/util/OpenLink.kt

```diff
@@ -0,0 +1,154 @@
+package com.jerboa.util
+
+import android.content.Context
+import android.content.Intent
+import android.net.Uri
+import android.widget.Toast
+import androidx.browser.customtabs.CustomTabsIntent
+import androidx.navigation.NavController
+import com.jerboa.R
+import com.jerboa.api.API
+
+enum class BrowserType(val id: Int, val resourceId: Int) {
```

Comment:
Looks like enums are supported by Room as of 2.3.0: https://developer.android.com/jetpack/androidx/releases/room#version_230_3

It'll save/load from strings, but we can instead use a converter to keep it as a number if we want: https://stackoverflow.com/questions/57326789/how-to-save-enum-field-in-the-database-room

So we can stick this with the rest of the db stuff instead of here.